### PR TITLE
04 alvis franks

### PR DESCRIPTION
**Pull request by zoyazip**

### CODE REVIEW

File: Majasdarbs/04-majasdarbs/cv-css/alvisfranks.html [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      <!-- CONTACT ME -->
+      <section id="contact-me">
+        <h2>Contact me:</h2>
+        <form action="submit_form.php">
+          <input
+            type="text"
+            id="firstname"
+            placeholder="Your name.."
+            name="firstname"
+            style="width: 200px"
```
Comment (zoyazip) [inline Majasdarbs/04-majasdarbs/cv-css/alvisfranks.html:179]: Could do this in css file

File: Majasdarbs/04-majasdarbs/cv-css/alvisfranks.html [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+          />
+          <br /><br />
```
Comment (zoyazip) [inline Majasdarbs/04-majasdarbs/cv-css/alvisfranks.html:181]: try to avoid br tag. Use margin or paddings instead

**Review by zoyazip — COMMENTED** [review]
good

File: Majasdarbs/04-majasdarbs/cv-css/css/style.css [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+  hr{
+    max-width: 900px;
+  }
+  address{
+    max-width: 900px;
+  }
+  body {
+    background-color: #252422;
+    color: rgb(189, 189, 189);
+    font-family: Consolas,monaco,monospace;
```
Comment (ruback105) [inline Majasdarbs/04-majasdarbs/cv-css/css/style.css:20]: You should set only imported fonts, don't see where you have imported these ones.

File: Majasdarbs/04-majasdarbs/cv-css/css/style.css [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+    font-family: 'Bebas Neue', cursive;
+    font-size: 80px;
+    flex-wrap: nowrap;
+    text-align: left;
+    display: flex;
+    margin: 0;
+    color: #fff;
+  }
+  #hero-title-position{
+    font-family: 'Bebas Neue', cursive;
```
Comment (ruback105) [inline Majasdarbs/04-majasdarbs/cv-css/css/style.css:38]: better to set this font to body (as I understand you are using it as a default font). And if you need some different font for some specific element, then import and use it.

File: Majasdarbs/04-majasdarbs/cv-css/css/style.css [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+  }
+  body {
+    background-color: #252422;
+    color: rgb(189, 189, 189);
+    font-family: Consolas,monaco,monospace;
+    margin: auto;
+  }
+  h1 {
+    font-weight: 700;
+    font-size: 2rem;
```
Comment (ruback105) [inline Majasdarbs/04-majasdarbs/cv-css/css/style.css:25]: you are using different units for font-sizes. Better use rems everywhere (not just here)

File: Majasdarbs/04-majasdarbs/cv-css/css/style.css [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+    font-size: 80px;
+    flex-wrap: nowrap;
+    text-align: left;
+    display: flex;
+    margin: 0;
+    color: #fff;
+  }
+  #hero-title-position{
+    font-family: 'Bebas Neue', cursive;
+    font-size: 300%;
```
Comment (ruback105) [inline Majasdarbs/04-majasdarbs/cv-css/css/style.css:39]: don't use percentages for font size, that's a bad idea

File: Majasdarbs/04-majasdarbs/cv-css/css/style.css [hunk 7]
```diff
@@ -0,0 +1,159 @@
...
+    justify-content: center;
+    font-family: 'Bebas Neue', cursive;
+    font-size: 50px;
+    text-decoration: underline;
+  }
+  
+  /* HOVERS */
+  #right-container a:hover,
+  header a:hover{
+    color: #a72f00;
```
Comment (ruback105) [inline Majasdarbs/04-majasdarbs/cv-css/css/style.css:66]: you can also add transition to color change, it looks smoother that way

File: Majasdarbs/04-majasdarbs/cv-css/css/style.css [hunk 8]
```diff
@@ -0,0 +1,159 @@
...
+  header ul {
+    display: flex;
+    padding-right: 20px;
+    flex-direction: row;
+    justify-content: space-between;
+    list-style: none;
+  }
+  header{
+    position: sticky;
+    margin: 0 0;
```
Comment (ruback105) [inline Majasdarbs/04-majasdarbs/cv-css/css/style.css:114]: you can use just margin: 0

**Pull request closed without merging**